corporate_partner_access/api/v1/serializers.py

```diff
@@ -40,7 +40,7 @@ class CorporatePartnerSerializer(serializers.ModelSerializer):
     class Meta:
         model = CorporatePartner
         fields = ["id", "code", "name", "homepage_url", "logo", "logo_url"]
-        read_only_fields = ["id", "logo_url"]
+        read_only_fields = ["id", "logo_url", "code"]
```

Comment:
I think you should remove the read_only_fields code if we're not going to auto-generate it yet.